### PR TITLE
Feature/git buildpackage

### PR DESCRIPTION
**Pull request by mvo5**
Tiny branch that allows us to use "git-buildpackage" (gbp buildpackage) to build a deb out of snappy. Along the way I removed .bzrignore and .bzr-buildpackage/.

### CODE REVIEW

**Comment by niemeyer** [comment]
Looks good to me.

**Comment by sergiusens** [comment]
Do you still want recipe building or is this not affected?

**Comment by mvo5** [comment]
That is not affected (should be independent)

File: .gitignore [hunk 1]
```diff
@@ -1,5 +1,4 @@
 _integration-tests/bin/
 _integration-tests/data/output/
-./share
-./snappy/hello_1.0.1_all.snap
-./tags
+share
+tags
```
Comment (elopio) [inline .gitignore]: Can you please add here .coverage ?

Comment (mvo5) [inline .gitignore]: Thanks, I added that now.

**Comment by mvo5** [comment]
I added the .coverage to git-buildpackage now

**Comment by chipaca** [comment]
Good.

**Pull request merged**